cmd/go-vcs/go-vcs.go

```diff
@@ -14,9 +14,10 @@ import (
 	"strings"
 
 	"github.com/kr/text"
+	vcs2 "github.com/shurcooL/go/vcs"
 	"sourcegraph.com/sourcegraph/go-diff/diff"
 	"sourcegraph.com/sourcegraph/go-vcs/vcs"
-	_ "sourcegraph.com/sourcegraph/go-vcs/vcs/git"
+	//_ "sourcegraph.com/sourcegraph/go-vcs/vcs/git"
```

Comment:
Can you add a TODO about this? Or even better, a GitHub issue or list in the readme of "differences between gitcmd and git impls".